package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@clutter/wt",
-  "version": "3.0.0-alpha.1",
+  "version": "3.0.0-alpha.2",
```

Comment:
Should we go straight to v3.0.0 to match release title? I'm not sure what the semantic versioning is here.

Reply:
ah yes - this was just for testing purposes, i'll update to 3.0.0 prior to merging